### PR TITLE
Post statii UI

### LINKED ISSUE #305: Author role has some non-intuitive results
As author role, there are two user interface functions with non-intuitive expectations:
1. "Delete" link in Text > Organize > Edit column will appear to delete another authors’ content and will not provide the same permissions error message "You are not allowed to edit this item.” found when selecting edit on another authors’ content. The message "Are you sure you want to delete this?” appears but invoking does nothing because I am not the author of the content. Expectation was a permissions alert.
2. Authors can re-arrange order of matter in Text > Organize   including other authors’ matter. Expectation was that I can only arrange my own content.

### LINKED ISSUE #1025: Change "Language Support" to "Language and Script Support"
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing this issue with one of our built-in root or book themes?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Description

The _Language Support_ section should be changed to _Language and Script Support_, and Hindi should be changed to Devanagari (as Devanagari supports Hindi, Sanskrit, and over 120 other languages). See https://github.com/pressbooks/pressbooks/issues/1023.

### Steps to Reproduce

Not applicable.

### System Information

Pressbooks: 5.0-dev
WordPress: 4.9.1
PHP: 7.1

### PR DESCRIPTION
**Pull request by greatislander**
- [x] Remove `export-only` post status in favour of core `private` post status
- [x] Update Language Support copy (fix #1025)
- [x] Add new post statii controls to Organize page
- [x] Add keyboard accessible reordering to Organize page
- [x] Clean up mobile view on Organize page
- [x] Add screen reader support to Organize page
- [x] Add new post statii controls to Edit page
- [x] Move Ebook start point to Theme Options, migrate value (#1053)
- [x] Consolidate API calls from Organize page using admin AJAX (avoid multiple HTTP calls)
- [x] Handle full-column updates on Organize page using new hooks
- [x] Disable drag and drop reordering on the Organize page if user lacks permissions (fix #305)
- [x] Ensure user without publish permissions cannot publish content (fix #305)

### CODE REVIEW

File: inc/modules/export/epub/templates/epub201/opf.php [hunk 1]
```diff
@@ -174,10 +174,16 @@
...
+		$ebook_options = get_option( 'pressbooks_theme_options_ebook', [] );
+		if ( isset( $ebook_options['ebook_start_point'] ) && ! empty( $ebook_options['ebook_start_point'] ) ) {
+			$start_id = $ebook_options['ebook_start_point'];
+		}
+
+		// Determine the manifest key for the post ID.
+		foreach ( $manifest as $key => $value ) {
+			if ( $start_id === $value['ID'] ) {
 				$start_key = $key;
-				break;
```
Comment (connerbw) [inline inc/modules/export/epub/templates/epub201/opf.php]: Any reason that the break was removed? if ` $manifest` has 10,000 items, and `$start_id === $value['ID']` on the third loop, then without the `break` the loop will keep running another 9997 times.

File: inc/modules/export/epub/templates/epub3/opf.php [hunk 2]
```diff
@@ -176,10 +176,16 @@
...
+		$ebook_options = get_option( 'pressbooks_theme_options_ebook', [] );
+		if ( isset( $ebook_options['ebook_start_point'] ) && ! empty( $ebook_options['ebook_start_point'] ) ) {
+			$start_id = $ebook_options['ebook_start_point'];
+		}
+
+		// Determine the manifest key for the post ID.
+		foreach ( $manifest as $key => $value ) {
+			if ( $start_id === $value['ID'] ) {
 				$start_key = $key;
-				break;
```
Comment (connerbw) [inline inc/modules/export/epub/templates/epub3/opf.php]: Keep the break.

**Comment by greatislander** [comment]
TODO: use admin AJAX instead of Backbone WP API so we can make batch calls.

**Comment by connerbw** [comment]
The version of this feature currently on Pressbooks.com (inconsistently 😢) changes the cursor into a hand. This version doesn't seem to do that.

css-missing-pointer

Otherwise looks OK to me.

File: inc/admin/organize/namespace.php [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+function update_post_visibility() {
+	if ( check_ajax_referer( 'pb-organize-visibility' ) ) {
+		$post_ids = explode( ',', $_POST['post_ids'] );
+		$format = ( isset( $_POST['export'] ) ) ? 'export' : 'web';
+		$visibility = absint( $_POST[ $format ] );
+
+		if ( current_user_can( 'publish_posts' ) ) {
+			remove_action( 'save_post', '\Pressbooks\Admin\Metaboxes\publish_fields_save' );
+			foreach ( $post_ids as $post_id ) {
+				$current_status = get_post_status( $post_id );
```
Comment (connerbw) [inline inc/admin/organize/namespace.php]: `$post_id = (int) $post_id; // Paranoia reasons`

File: inc/admin/organize/namespace.php [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+ * @since 5.0.0
+
+ */
+function update_post_title_visibility() {
+	if ( check_ajax_referer( 'pb-organize-showtitle' ) ) {
+		$post_ids = explode( ',', $_POST['post_ids'] );
+		$pb_show_title = $_POST['show_title'];
+
+		foreach ( $post_ids as $post_id ) {
+			if ( current_user_can( 'edit_post', $post_id ) ) {
```
Comment (connerbw) [inline inc/admin/organize/namespace.php]: `$post_id = (int) $post_id; // Paranoia reasons`

**Pull request merged**